components/NewsItem.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+	<!-- This compenent is used by data from pages api call as well as post api call -->
+	<nuxt-link :to="'/news-and-views/' + slug" class="news-item">
+		<figure class="preview-media">
+			<img :src="picture" class="news-item-img">
+			<figcaption>
+				<h2>
+					{{ title }}
+				</h2>
+			</figcaption>
+			<div class="date">
+				<img class="date-icon">
+				<time>{{ date }}</time>
+			</div>
+		</figure>
+	</nuxt-link>
+</template>
+
+<script>
+export default {
+	// Pages have no media attached to them so when they render they are imageless
```

Comment:
I'm a bit confused about this comment, or maybe what's expected. It says that the pages render imageless, but it looks like we add a default picture below.

Reply:
This should be removed now. Was added before default images were implemented.